packages/calypso-codemods/transforms/replace-lodash-get.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Transform all safely recognizable instances of _.get().
+ * Conditions include:
+ * - Must be imported in the form `import { get, ... } from 'lodash'` or `import { get as foo, ... } from 'lodash'`
+ * - Must either not use a default parameter, or use a default parameter set to the `null` literal
+ * - Must use a string literal, string template, or array expression as the path
+ *
+ * If at least one of the conditions is not met, the transformation will be dropped.
+ *
+ * In files with at least one successful transformation, the import is removed if no longer used.
+ */
+
+// Transform function.
+export default function transformer( file, api ) {
+	const j = api.jscodeshift;
+	const root = j( file.source );
+
+	let getMethodName;
+	let visitedGetCalls = 0;
+	let modifiedGetCalls = 0;
+
+	function debug( text ) {
+		throw new Error( `${ file.path }: ${ text }` );
+	}
+
+	function isValidIdentifier( string ) {
+		return string.match( /^[A-Za-z]\w*$/ );
```

Comment:
Noting that an identifier could also start with an underscore.

Reply:
Good catch, thank you!